src/pages/Room/Calendar.jsx

```diff
@@ -9,79 +9,70 @@ import { withAuthContext } from "hocs";
 import { NotificationProvider, ModalFormProvider } from "providers";
 import { NoteCard } from "components/NoteCard";
 
-class CalendarComponent extends React.Component {
-  state = {
-    isDBEmpty: false,
-    isDrawerOpen: false,
-    isLoading: true
-  };
+const CalendarPage = props => {
+  const [isDBEmpty, setIsDBEmpty] = React.useState(false);
+  const [isDrawerOpen, setIsDrawerOpen] = React.useState(false);
+  const [isLoading, setIsLoading] = React.useState(true);
 
-  handleDrawerOpen = () => {
-    this.setState({ isDrawerOpen: true });
+  const handleDrawerOpen = () => {
+    setIsDrawerOpen(true);
   };
 
-  handleDrawerClose = () => {
-    this.setState({ isDrawerOpen: false });
+  const handleDrawerClose = () => {
+    setIsDrawerOpen(false);
   };
 
-  handleDBEmpty = () => {
-    this.setState({ isDBEmpty: true });
+  const handleDBEmpty = () => {
+    setIsDBEmpty(true);
   };
 
-  componentDidMount() {
-    this.setState({ isLoading: false });
+  React.useEffect(() => {
+    setIsLoading(false);
+  }, []);
+
+  const { authContext, bookingsData, onBookingsDataChange, URLRoomId } = props;
+  const { isAuth } = authContext;
+
+  if (!isAuth && !isLoading) {
+    return <Redirect to="/login" />;
   }
 
-  render() {
-    const { isDBEmpty, isDrawerOpen, isLoading } = this.state;
-    const {
-      authContext,
-      bookingsData,
-      onBookingsDataChange,
-      URLRoomId
-    } = this.props;
-    const { isAuth } = authContext;
-    if (!isAuth && !isLoading) {
-      return <Redirect to="/login" />;
-    }
+  const { sessionInfo } = authContext;
+  return (
+    <Fragment>
+      <NavBar authContext={authContext} />
+      <NotificationProvider>
+        <ModalFormProvider onBookingsDataChange={onBookingsDataChange}>
+          <Grid container direction="row">
+            <DrawerBookings
+              isOpen={isDrawerOpen}
+              handleOpen={handleDrawerOpen}
+              handleClose={handleDrawerClose}
+            >
+              <BookingsSideBar
+                bookingsData={bookingsData}
+                onBookingsDataChange={onBookingsDataChange}
+              />
+            </DrawerBookings>
+            <Grid item xs={12}>
+              {isDBEmpty && <NoteCard />}
 
-    const { sessionInfo } = authContext;
-    return (
-      <Fragment>
-        <NavBar authContext={authContext} />
-        <NotificationProvider>
-          <ModalFormProvider onBookingsDataChange={onBookingsDataChange}>
-            <Grid container direction="row">
-              <DrawerBookings
-                isOpen={isDrawerOpen}
-                handleOpen={this.handleDrawerOpen}
-                handleClose={this.handleDrawerClose}
-              >
-                <BookingsSideBar
+              {!isDBEmpty && (
```

Comment:
It's not better to left the ternary operator? I think that clarifies the idea that the DB is empty

Reply:
I prefer that style of conditional rendering, nested ternary tend to get messy